crates/wash-lib/src/cli/mod.rs

```diff
@@ -329,16 +329,16 @@ pub fn extract_keypair(
     }
 }
 
-/// Transforms a list of labels in the form of (label=value) to a hashmap
-pub fn labels_vec_to_hashmap(constraints: Vec<String>) -> Result<HashMap<String, String>> {
+/// Transforms a list of key in the form of (key=value) to a hashmap
+pub fn input_vec_to_hashmap(values: Vec<String>) -> Result<HashMap<String, String>> {
```

Comment:
Is this going to cause an issue for `wash label`?

Reply:
Looks like `wash label` does a custom parse instead of using this function 🤔